auth-web/src/components/auth/staff/account-management/StaffPendingAccountsTable.vue

```diff
@@ -1,57 +1,120 @@
+                        <v-text-field class="text-input-style" append-icon="mdi-calendar" dense filled hide-details="true"
+                        :placeholder="header.text" v-model="dateTxt" />
+                      </div>
+
+                      <v-btn v-else-if="searchParamsExist && header.value === 'action'" outlined color="primary"
+                        class="action-btn clear-filter-button" @click="clearSearchParams()">
+                        <span class="clear-filter cursor-pointer">
+                          Clear Filters
+                          <v-icon small color="primary">mdi-close</v-icon>
+                        </span>
+                      </v-btn>
+                    </th>
+                  </tr>
+                </thead>
+              </template>
+
+              <template v-slot:[`item.dateSubmitted`]="{ item }">
+                {{ formatDate(item.dateSubmitted, 'MMM DD, YYYY') }}
+              </template>
+              <template v-slot:[`item.type`]="{ item }">
+                {{ item.relationshipType === TaskRelationshipTypeEnum.PRODUCT ? `Access Request (${item.type})` :
+                    item.type
+                }}
+              </template>
+              <template v-slot:[`item.status`]="{ item }">
+                <span class="status" :class="{ 'onhold': item.status == 'HOLD' }">{{ item.status == 'HOLD' ? `On hold` :
+                    item.status.toLowerCase()
+                }}</span>
+              </template>
+              <template v-slot:[`item.action`]="{ item }">
+                <div class="btn-inline">
+                  <v-btn outlined color="primary" :data-test="getIndexedTag('reset-password-button', item.id)"
+                    @click="review(item)">
+                    Review
+                  </v-btn>
+                </div>
+              </template>
+            </v-data-table>
+          </transition>
+        </v-col>
+      </v-row>
+    </v-form>
+  </v-container>
 </template>
 
 <script lang="ts">
 import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { SessionStorageKeys, TaskRelationshipStatus, TaskRelationshipType, TaskStatus } from '@/util/constants'
 import { Task, TaskFilterParams, TaskList } from '@/models/Task'
-import { TaskRelationshipStatus, TaskRelationshipType, TaskStatus } from '@/util/constants'
+import { mapActions, mapState } from 'vuex'
 import CommonUtils from '@/util/common-util'
+import ConfigHelper from '@/util/config-helper'
 import { DataOptions } from 'vuetify'
+import DatePicker from '@/components/auth/common/DatePicker.vue'
 import PaginationMixin from '@/components/auth/mixins/PaginationMixin.vue'
+import { ProductCode } from '@/models/Staff'
+import moment from 'moment'
 import { namespace } from 'vuex-class'
 
 const TaskModule = namespace('task')
 
 @Component({
+  components: {
+    DatePicker
+  },
+  computed: {
+    ...mapState('staff', ['products'])
```

Comment:
There's a better way to do this, but it's all getting axed anyways

Reply:
followed the same pattern in other components in auth web

Reply:
There are two patterns in auth web :(

Reply:
```
  @ActivityLogModule.State('currentOrgActivity') public currentOrgActivity!: ActivityLog
  @ActivityLogModule.Action('getActivityLog') public getActivityLog!:(filterParams:ActivityLogFilterParams) =>Promise<ActivityLog>
  ```